comment/forms.py

```diff
@@ -1,10 +1,40 @@
 from django import forms
+from django.utils.translation import gettext_lazy as _
 
 from comment.models import Comment
+from comment.conf import settings
 
 
 class CommentForm(forms.ModelForm):
     class Meta:
         model = Comment
-        fields = ('content', )
+        fields = ('content', 'email')
```

Comment:
The old approach seems to be better. I don't like unnecessary hidden field. Construct the field on `if self.request.user.is_anonymous and settings.COMMENT_ALLOW_ANONYMOUS` is true.

Reply:
Yes, seems legit.